README.md

```diff
@@ -137,6 +137,9 @@ to automatically deploy to a staging repository.
 publishing {
     repositories {
         sonatypeMavenCentral {
+            // If you need to publish to a different repository
+            url = "https://s01.oss.sonatype.org/service/local/"
```

Comment:
the docs talk about a `snapshotRepositoryUrl` too - is that relevant here ?

```
nexusUrl.set(uri("https://s01.oss.sonatype.org/service/local/"))
snapshotRepositoryUrl.set(uri("https://s01.oss.sonatype.org/content/repositories/snapshots/"))
```

Reply:
No, because this repository is not used for snapshots. You'll need to use `sonatypeSnapshots()` for that.

Maybe I should mention that too?

Reply:
the nexus-publishing-plugin automatically selected the correct repository ? we don't do that and still manually switch between `sonatypeMavenCentral` and `sonatypeSnapshots` ?

Reply:
That is correct, we hardcode to always use the staging repository.

Personally, I find it confusing that the switching happens automatically, and my preferences have influenced the initial design of the plugin.